analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RoslynSymbolicExecution.cs

```diff
@@ -261,7 +267,7 @@ operation.Kind switch
             operation.Instance.Kind switch
             {
                 OperationKindEx.Invocation => ExceptionState.UnknownException,
-                // ToDo: Support other operations like field/property access on non-static non-this, conversions and so on. See docs for list of operations.
```

Comment:
This change is not needed and will cause rebase issues with my PR.

Reply:
That wasn't intentional :/